packages/components/src/Chip/Chip.css

```diff
@@ -0,0 +1,96 @@
+:root {
+  --chip-height: 40px;
+  --chip-radius: 20px;
+}
+
+.chip {
+  display: flex;
+  position: relative;
+  height: var(--chip-height);
+  padding: var(--space-small) var(--base-unit);
+  border: none;
+  border-radius: var(--chip-radius);
+  text-align: left;
+  cursor: pointer;
+  transition: all ease-in-out 0.12s;
+  align-items: center;
+}
+
+.chip:focus {
+  box-shadow: var(--shadow-focus);
+}
+
+.chip .prefix {
+  margin-left: calc(-1 * var(--space-small));
+  margin-right: var(--space-small);
+}
+
+.chip .suffix {
+  margin-left: var(--space-small);
+  margin-right: calc(-1 * var(--space-small));
+}
+
+.chip .prefix,
+.chip .suffix {
+  display: flex;
+  width: var(--space-large);
+  height: var(--space-large);
+  border-radius: var(--radius-circle);
+  background-color: var(--color-white);
+  align-items: center;
+  justify-content: center;
+}
+
+.chip .prefix.empty,
+.chip .suffix.empty {
+  display: none;
+}
+
+.chip .chipBar {
+  width: var(--space-minuscule);
+  height: var(--base-unit);
+  margin: 0 var(--space-small);
+  background-color: var(--color-greyBlue--light);
+}
+
+.chip .selected {
+  border-radius: var(--radius-circle);
+  background-color: var(--color-white);
+}
+
+.base {
+  color: var(--color-blue);
+  background-color: var(--color-greyBlue--lightest);
+}
+.base:hover,
+.base:focus {
+  background-color: var(--color-greyBlue--lighter);
+}
+
+.invalid,
+.invalid:hover,
+.invalid:focus,
+.subtle.invalid,
+.subtle.invalid:hover,
+.subtle.invalid:focus {
+  border: var(--border-base) solid var(--color-red);
+  background-color: var(--color-red--lightest);
+}
+
+.subtle {
+  border: var(--border-base) solid var(--color-grey--lighter);
+  background-color: var(--color-white);
+}
+
+.disabled,
+.disabled:hover,
+.disabled:focus {
+  color: var(--color-grey);
+  background-color: var(--color-grey--lighter);
+  cursor: auto;
+}
+
+.invalid.disabled {
+  pointer-events: none;
+  opacity: 0;
```

Comment:
I think this is where the invisible variants are coming from!

Reply:
I wasn't 100% what to do here. In your chart, when things are both invalid and disabled at the same time, they don't show up but in your figma they still take up space. I thought that might be weird to just mark them with display:none, but I can definitely do that if you would prefer.

![image](https://github.com/GetJobber/atlantis/assets/145565743/b4155a1f-ffa5-40e9-8ab1-f4f53ea1cf3b)

Reply:
Ah! What that's indicating is that a chip that's both `invalid` and `disabled` shouldn't exist, as it offers the user no recourse to fix whatever issue has led to the invalid state. I do wonder if we need to _technically_ exclude those variations, or just leave it (maybe it just gets the disabled styling?) and if a team designs their way into that state, we can have guidance on why they shouldn't?

Reply:
We could mark them as `display:none` which does effectively remove them from the DOM screen readers, all of that, but it will give a clue to a developer that their thing _is rendering_ but that it's not showing up because they misconfigured it.

Or we could code it do just flat out not render anything in that situation, but then we run into the "why doesn't my component show up."

I think writing in the documentation to not use invalid/disabled at the same time, and by hiding in the DOM should be enough. I'm not huge on introducing code to print warnings for developers (unless it's potentially destructive) because that runs the risk of something going weird in Prod for a hand-hold.